tools/clang/unittests/HLSLExec/ExecutionTest.cpp

```diff
@@ -11096,10 +10309,6 @@ TEST_F(ExecutionTest, PackUnpackTest) {
   }
 }
 
-// This test expects a <pShader> that retrieves a signal value from each of a
-// few resources that are initialized here. <isDynamic> determines if it uses
```

Comment:
Why was this comment removed?

Reply:
Accidental during manual merge, will restore.